README.md

```diff
@@ -113,6 +113,46 @@ docker run --rm \
   plugins/docker --dry-run
 ```
 
+### GAR (Google Artifact Registry)
+
+```yaml
+kind: pipeline
+name: default
+type: docker
+
+steps:
+  - name: push-to-gar
+    image: plugins/gcr
+    pull: never
+    settings:
+      tag: latest
+      repo: project-id/repo/image-name
+      registry_type: GAR
```

Comment:
this should go into plugins.drone.io as well

Reply:
will do.